pkg/volume/flexvolume/common_test.go

```diff
@@ -19,12 +19,12 @@ package flexvolume
 import (
 	"encoding/json"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/kubernetes/pkg/volume"
 	volumetesting "k8s.io/kubernetes/pkg/volume/testing"
 	"k8s.io/kubernetes/test/utils/harness"
 	"k8s.io/utils/exec"
-	"k8s.io/utils/exec/testing"
+	testingexec "k8s.io/utils/exec/testing"
```

Comment:
I'm confused since no references to this import seem to be affected.

Anyway, if this is disambiguating with "k8s.io/kubernetes/pkg/volume/testing" then the alias ought to be "exectesting"